tox.ini

```diff
@@ -1,14 +1,23 @@
 [tox]
 
 envlist =
-    flake8
-    mypy
-
+    flake8, black, mypy
     coverage-py{27,35,36,37,38,py2,py3}-tw{171,184,current,trunk}
-
+    coverage_report
     docs, docs-linkcheck
+    packaging
+
+skip_missing_interpreters = {tty:True:False}
+
+
+[default]
+
+basepython = python3.8
+
+setenv =
+    PY_MODULE=klein
 
-skip_missing_interpreters = {env:TOX_SKIP_MISSING_INTERPRETERS:True}
+    PYTHONPYCACHEPREFIX={envtmpdir}/pycache
```

Comment:
Write `__pycache__` in `./tox/` to reduce the number of build artifacts left in the source tree.